salt/django.sls

```diff
@@ -77,21 +77,28 @@ django-deps:
   virtualenv.managed:
     - python: /usr/bin/python3
     - user: {{ pillar.user }}
-    - requirements: {{ djangodir }}requirements.txt
     - require:
       - pkg: django-deps
       - git: {{ pillar.git.url }}{{ djangodir }}
       - file: set_lc_all # required to avoid unicode errors for the "schema" library
     - watch_in:
       - service: apache2
 
+pip_install_requirements:
```

Comment:
Please add a comment:

```python
# Due to a bug in Salt, we can't use `- requirements: {{ djangodir }}requirements.txt` in the above
# `{{ djangodir }}.ve/` state. See https://github.com/saltstack/salt/issues/56514
```

That way, once the bug is fixed, we can revert to a simpler Salt file.